package.json

```diff
@@ -47,7 +49,7 @@
     "style-loader": "^3.3.1",
     "ts-loader": "^9.4.1",
     "typescript": "^4.8.4",
-    "typescript-plugin-css-modules": "^3.4.0",
+    "typescript-plugin-css-modules": "^2.8.0",
```

Comment:
why do you downgrade this dependency?

Reply:
It was because I run the command` npm audti fix --force`